pkgs/applications/editors/emacs/elisp-packages/manual-packages/lspce/module.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  callPackage,
+  emacsPackages,
+  fetchFromGitHub,
+  rustPlatform,
+}:
+
+let
+  self = rustPlatform.buildRustPackage {
+    pname = "lspce-module";
+    version = "1.1.0-unstable-2024-07-13";
```

Comment:
Can we still use `unstableGitUpdater` to auto-update?

Reply:
I don't think so. The updaters are not so smart to look outside their files, I believe.

But maybe we can modify the updaters to do something like "hey, don't look at me, look to this file, the hashes are there".